### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by annguyen23**

### CODE REVIEW

File: 2020-03-Autumn/3_backtracking/README.md [hunk 1]
```diff
@@ -58,21 +58,37 @@ Go to [Solution](#s1)   [Top](#top)
 <!-- Don't remove -->
 <a name="p2"/>
 
-### 2. PROBLEM 2 TODO :bug:
+### 2. PROBLEM 2 Subsets
```
Comment (mduw) [inline 2020-03-Autumn/3_backtracking/README.md:61]: Remove problem 2 from title ==>  ### 2. Subsets

File: 2020-03-Autumn/3_backtracking/README.md [hunk 2]
```diff
@@ -58,21 +58,37 @@ Go to [Solution](#s1)   [Top](#top)
...
 
-Source: TODO :bug:
+Source: [LeetCode](https://leetcode.com/problems/subsets/)
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Given an integer array nums, return all possible subsets (the power set).
```
Comment (mduw) [inline 2020-03-Autumn/3_backtracking/README.md:67]: you're missing important an important constraint: 
> The solution set must not contain duplicate subsets. Return the solution in any order.

if you decide to change the problem statement, pls change your solution accordingly.

File: 2020-03-Autumn/3_backtracking/README.md [hunk 3]
```diff
@@ -58,21 +58,37 @@ Go to [Solution](#s1)   [Top](#top)
...
+
+1 <= nums.length <= 10
+-10 <= nums[i] <= 10
+All the numbers of nums are unique.
 
 #### Function Signature
 
-TODO :bug:
+C++: vector<vector<int>> subsets(vector<int>& nums)
+Java: public List<List<Integer>> subsets(int[] nums)
```
Comment (mduw) [inline 2020-03-Autumn/3_backtracking/README.md:91]: pls use inline code or code block for it

File: 2020-03-Autumn/3_backtracking/README.md [hunk 4]
```diff
@@ -206,19 +222,44 @@ Go to [Top](#top)
...
+        for (int i = index; i < nums.length; i++) {
+            // call recursion (backtracking) method
+            helper (result, current, nums, i+1);
+        }
+        
+        // remove index-th number from current list
+        current.remove (current.size()-1);
+        
+    }
+}
```
Comment (mduw) [inline 2020-03-Autumn/3_backtracking/README.md:262]: pls use code block for it

**Review by mduw — CHANGES_REQUESTED** [review]
pls place your solution in `2021-01-Winter` NOT '2020-03-Autumn`

File: 2020-03-Autumn/3_backtracking/README.md (hunk 3 above)
Comment (annguyen23) [reply 2020-03-Autumn/3_backtracking/README.md:91]: How do I change it to code block?

File: 2020-03-Autumn/3_backtracking/README.md (hunk 4 above)
Comment (annguyen23) [reply 2020-03-Autumn/3_backtracking/README.md:262]: How do I change it to code block?

**Comment by mduw** [comment]
Please provide a short description for your solution. i dont see it in this PR

File: 2020-03-Autumn/3_backtracking/README.md (hunk 3 above)
Comment (mduw) [reply 2020-03-Autumn/3_backtracking/README.md:91]: use \' \' for inline code
use \``` \``` for code block

https://guides.github.com/features/mastering-markdown/#syntax

**Pull request closed without merging**